cmd/monaco/download/download_configs.go

```diff
@@ -256,6 +256,18 @@ func downloadConfigs(downloaders downloaders, opts downloadConfigsOptions) (proj
 		}
 	}
 
+	{
+		if opts.auth.OAuth != nil {
+			bucketCfgs, err := downloaders.Bucket().Download(opts.projectName)
+			if err != nil {
+				return nil, err
+			}
+			copyConfigs(configs, bucketCfgs)
+		} else if opts.onlyAutomation {
```

Comment:
that `else if` on the `onlyAutomation` flag does not make sense here. If we have no explicit way to request bucket downloads, I think it's enough to just do nothing if no oauth is configured